index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+    integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous" />
+  <link rel="stylesheet" href="style.css">
+  <title>Yuriy's Portfolio</title>
+</head>
+
+<body>
+  <div id="container">
+    <nav id="navigation">
+      <span id="logo">My Logo</span>
+      <ul id="menu">
+        <li class="menu-item big">Portfolio</li>
+        <li class="menu-item big">About</li>
+        <li class="menu-item big">Contact</li>
+        <li class="menu-item" id="hidden"><i class="fas fa-bars"></i></li>
+      </ul>
+    </nav>
+    <section id="bottom">
+      <header id="main">
+        <h2 class="title">I'm Yuriy</h2>
```

Comment:
- [ ] I think is better if you use a `h1` for the main title of you're web app, this more [semantically adequate](https://www.w3schools.com/html/html5_semantic_elements.asp) and has good effects in the SEO of your portfolio.

- [ ] Try to stick as much as possible to the design provided. I can see some of the colors are missing from your text and icons. You can change the color of these with the `color` property in your stylesheet. For example, the title has the following color in the template: `#172B4D` 

![image](https://user-images.githubusercontent.com/79114233/138187815-ace5c0df-be21-4519-aeb6-bb4555120ce0.png)

![image](https://user-images.githubusercontent.com/79114233/138187875-a364f207-b083-4706-a848-f9c02f57d563.png)